week-4/F-object-keys/exercise-part-0.js

```diff
@@ -0,0 +1,32 @@
+/*
+
+Return the keys of the following object
+
+*/
+
+var capitalCities = {
+  edinburgh: 'Scotland',
+  nairobi: 'Kenya',
+  canberra: 'Australia',
+  ottowa: 'Canada'
+};
+
+var highScores = {
+  55: 'Alistair',
+  100: 'David',
+  89: 'Hannah',
+  34: ['Sergi', 'Frank',]
+};
+
+// ONLY EDIT BELOW HERE
+
+var capitalCitiesKeys = ;
+var highScoresKeys;
+
+// ONLY EDIT ABOVE HERE
+
+console.log(capitalCitiesKeys);
+// Should return [ 'edinburgh', 'nairobi', 'canberra', 'ottowa' ]
+
+console.log(highScoresKeys)
+// Should return ['34, '55', '89', '100']
```

Comment:
In the other lessons, I thought of adding the expected outputs in the console.log itself, so it stands out more to students if they made a mistake:

```js
console.log("Expected result: edinburgh, nairobi, canberra, ottowa. Actual result: " + capitalCitiesKeys);
```